ftplugin/python/SimpylFold.vim

```diff
@@ -116,6 +116,17 @@ function! s:NumContainingDefs(lnum)
 
 endfunction
 
+" Construct a foldexpr value
+function! s:FoldExpr(lnum, foldlevel)
```

Comment:
Yeah, makes sense for now, but I have a few more commits coming that will expand this function for caching purposes.